app/controllers/application_controller.rb

```diff
@@ -2,4 +2,26 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.for(:sign_up) << [
+      :first_name,
+      :last_name,
+      :address,
+      :city,
+      :state,
+      :zipcode
+      ]
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.